From 05ffe3749d73942cf4df7ed8f53ae239e62d5376 Mon Sep 17 00:00:00 2001
From: Jon Maloy <jmaloy@redhat.com>
Date: Tue, 25 Jun 2024 22:30:19 -0400
Subject: [PATCH 11/31] SecurityPkg/RngDxe: Rename AArch64/RngDxe.c

RH-Author: Jon Maloy <jmaloy@redhat.com>
RH-MergeRequest: 77: UINT32 overflow in S3 ResumeCount and Pixiefail fixes
RH-Jira: RHEL-21854 RHEL-21856 RHEL-40099
RH-Acked-by: Gerd Hoffmann <None>
RH-Commit: [11/31] 39c8a7a1e45c7c26f5d16f79d81abac1fbae4f22

JIRA: https://issues.redhat.com/browse/RHEL-21856
CVE: CVE-2022-45237
Upstream: Merged

commit ff29cdb968a1a4d7bd7ab4eba2597a77c0748dc2
Author: Pierre Gondois <pierre.gondois@arm.com>
Date:   Fri Oct 28 17:32:57 2022 +0200

    SecurityPkg/RngDxe: Rename AArch64/RngDxe.c

    To re-use the AArch64/RngDxe.c for an Arm implementation,
    rename AArch64/RngDxe.c to ArmRngDxe.c.

    Acked-by: Leif Lindholm <quic_llindhol@quicinc.com>
    Signed-off-by: Pierre Gondois <Pierre.Gondois@arm.com>
    Acked-by: Jiewen Yao <jiewen.yao@intel.com>

Signed-off-by: Jon Maloy <jmaloy@redhat.com>
---
 .../RngDxe/{AArch64/RngDxe.c => ArmRngDxe.c}                    | 0
 SecurityPkg/RandomNumberGenerator/RngDxe/RngDxe.inf             | 2 +-
 2 files changed, 1 insertion(+), 1 deletion(-)
 rename SecurityPkg/RandomNumberGenerator/RngDxe/{AArch64/RngDxe.c => ArmRngDxe.c} (100%)

diff --git a/SecurityPkg/RandomNumberGenerator/RngDxe/AArch64/RngDxe.c b/SecurityPkg/RandomNumberGenerator/RngDxe/ArmRngDxe.c
similarity index 100%
rename from SecurityPkg/RandomNumberGenerator/RngDxe/AArch64/RngDxe.c
rename to SecurityPkg/RandomNumberGenerator/RngDxe/ArmRngDxe.c
diff --git a/SecurityPkg/RandomNumberGenerator/RngDxe/RngDxe.inf b/SecurityPkg/RandomNumberGenerator/RngDxe/RngDxe.inf
index e0e767cbf3..1d0bdef57d 100644
--- a/SecurityPkg/RandomNumberGenerator/RngDxe/RngDxe.inf
+++ b/SecurityPkg/RandomNumberGenerator/RngDxe/RngDxe.inf
@@ -42,7 +42,7 @@
   Rand/AesCore.h
 
 [Sources.AARCH64]
-  AArch64/RngDxe.c
+  ArmRngDxe.c
   ArmTrng.c
 
 [Packages]
-- 
2.39.3

